sqlglot/optimizer/canonicalize.py

```diff
@@ -83,6 +84,16 @@ def remove_ascending_order(expression: exp.Expression) -> exp.Expression:
     return expression
 
 
+def expand_case_conditions(expression: exp.Expression) -> exp.Expression:
```

Comment:
do we need this? can this just go in simplify?

Reply:
could be moved yeah, will do

Reply:
yea, just do this in simplify

Reply:
Done in https://github.com/tobymao/sqlglot/pull/2403/commits/9a6481904aa1123a0d3c5ad71b0b2e47c883ae4b

Reply:
I also fixed a bug here: https://github.com/tobymao/sqlglot/pull/2403/commits/9a6481904aa1123a0d3c5ad71b0b2e47c883ae4b#diff-28f5a8e638b2f114cfd7411b4589283d786380e7f1cc040fd990c8290887b2dcR73